pybeautifyjson.fish

```diff
@@ -1,5 +1,7 @@
-# beautify json string
-# use : pybeautifyjson '{"foo": "lorem", "bar": "ipsum"}'
-function pybeautifyjson
-  echo $argv | _python -mjson.tool
+function pybeautifyjson -d "Beautify a JSON string"
```

Comment:
Much better! And allows to keep "namespacing" - `pyjsonpp`, which is an improvement over 10 chars.